pyproject.toml

```diff
@@ -37,7 +37,9 @@ dependencies = [
   "funcy",
   "gto",
   "ruamel.yaml",
-  "scmrepo"
+  "scmrepo",
+  "psutil",
+  "pynvml"
```

Comment:
Should we include by default or make them optional?

Reply:
Yep, my 2cs. I would prefer "system" to be enabled by default.  Agreed to check on how big / complicated those deps are.

Reply:
Both, when installed together, are quite light: ~500KB. 

@shcheklein Then should I change the `monitor_system` default to `True` in the `Live` object?
Any idea on that topic @dberenbaum?

My 0.02$ is that it would be nice to have it by default, especially for new users. But for existing users, it can be annoying to have new plots all over the studio dashboard I spend days customizing. What should we prioritize?

Reply:
> Then should I change the monitor_system default to True in the Live object?

yep, I would go for it. I think we should be optimizing for new users still. Not a strong opinion though.

Reply:
I would vote to keep it off and consider making it a default in 4.0 for the reasons @AlexandreKempf mentioned (annoyance to existing users)

Reply:
> off and consider making it a default in 4.0

sounds good. let's make a ticket for this? :) and do it soon I hope. (I just feel it's a really good and appealing feature - people will stick better with the extension, Studio, etc).

Reply:
Let's wait for 4.0. Do we have a roadmap to 4.0 by any chance, so I can add that ticket?
Also this ticket should mention that `psutil` and `pynvml` should not be optional anymore.

Reply:
If we make the deps optional, do we need to change the PR a bit to make sure we fail gracefully if they are not installed? It looks like loading `dvclive.live` now will try to import those deps.

Reply:
Ok, I put them by default as they are quite light. Also, having them installed by default should simplify the documentation.